packages/Radio/src/Radio.js

```diff
@@ -12,8 +12,9 @@ const propTypes = {
   isChecked: PropTypes.bool,
   isDisabled: PropTypes.bool,
   canDeselect: PropTypes.bool,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
   size: PropTypes.oneOf(ShirtSizes.DEFAULT),
+  value: PropTypes.shape({ id: PropTypes.string }).isRequired,
```

Comment:
Hm, how about:

```js
PropTypes.oneOfType([
  PropTypes.bool,
  PropTypes.string,
  PropTypes.number,
  PropTypes.shape({ id: PropTypes.string }),
]).isRequired
```

Reply:
This would require a bit of logic when you use the `value` as the `key`, but gives the consumer more flexibility.

Reply:
Yep, no longer using the value however as consumer can work with the `activeIndex` to get the data values